covid_model/cli_specs.py

```diff
@@ -14,7 +14,9 @@ def __init__(self):
         self.add_argument('-vprev', '--variant_prevalence', type=str)
         self.add_argument('-mprev', '--mab_prevalence', type=str)
         self.add_argument('-am', '--attribute_multipliers', type=str)
-        self.set_defaults(refresh_actual_vacc=False)
+        self.add_argument('-sd', '--start_date', type=str, help="format: YYYY-MM-DD")
+        self.add_argument('-ed', '--end_date', type=str, help="format: YYYY-MM-DD")
```

Comment:
if this borks things up, I can rethink it. Logically I think it should go here.

Reply:
I think this is a good addition. Some scripts are adding a from_date and to_date...I can do some cleanup on that so we don't have duplicate parameters.

The plotting scripts sometimes want to have a bounds for plotting that are different from the bounds for the model. Maybe a simple convention could be to call those plot_from_date and plot_to_date, for clarity.

Reply:
Yeah I thought all the "from_date" and "to_date" parameters were being used in plotting, but I could have missed some other uses. adding "plot_" makes sense.